SA/design_concept/content/design/non_functional_requirements.typ

```diff
@@ -3,7 +3,7 @@
 = Non Functional Requirements <non_functional_requirements>
 
 This section describes all non-functional requirements identified for VisualFP.
-To find relevant #ac("NFR")s, ISO-25010 @iso-25010 was used for inspiration.
+To find a relevant #ac("NFR"), ISO-25010 @iso-25010 was used for inspiration.
```

Comment:
This sounds like there is only a single non functional requirement

Reply:
The problem here is that the acronym explenation is between NFR and the s -> NFR (Non-Function Requirement)s, which looks weird.